src/components/basic/lists/AlternateListItem.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const AlternateListItem = props => (
```

Comment:
This should have a prop for `href` that is required (using proptypes). You should probably also have other link properties on this.

Reply:
Updated locally. Will commit after the tests run and pass.

Reply:
Committed initial proptypes.